deployment-examples/aws/k8s/values.yaml

```diff
@@ -0,0 +1,31 @@
+export TAG="0.0.1"
+
+image:
+  repository: $AWS_ECR_PATH/$AWS_ECR_REPOSITORY:$TAG
+  tag: '0.0.1'
+  pullPolicy: 'Always'
+
+## edit those environment variables
+## ----------------------
+nonSensitiveEnvs:
+  CTP_PROJECT_KEY: $CTP_PROJECT_KEY
+  CTP_API_URL: 'https://api.europe-west1.gcp.commercetools.com'
+  CTP_AUTH_URL: 'https://auth.europe-west1.gcp.commercetools.com'
+
+sensitiveEnvs:
+  CTP_CLIENT_ID: $CTP_CLIENT_ID
+  CTP_CLIENT_SECRET: $CTP_CLIENT_SECRET'
+## ----------------------
+
+# When the pod is scheduled, the container is guaranteed the amount of
+# resources requested 512MB RAM. As we don't know how much memory the jobs will
+# need, we don't set the limit
```

Comment:
The comment do not reflect what you have below. Did you try with the default memory?

Reply:
Yes, Even I copied this from some other repository. 
The comment says why we are not setting the resource limits. And yes here we set only memory for requests.
I found more clear document here about resource limits and memory.
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

I have updated the comment to make it more clear. 
https://github.com/commercetools/commercetools-subscriptions/pull/14/commits/023696e8895d503b39e269563ed6b86cd1be29a0

Reply:
Hmmm I'm not sure if you make it better. In the comment you have this:
```
we don't set the resource limit here
```
And then just below it you have this:
```
resources:
  requests:
    memory: '1024Mi'
```
I guess it's not a limit, but the job is requiring minimum of 1024Mi, but the comment is misleading. Please add a sentence explaining what does the `resources` mean below.

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/14/commits/4d087d55424dc72065c8fbbb4d5fb9a446379916